pyproject.toml

```diff
@@ -87,3 +87,8 @@ ignore-words-list = 'te'
 # TODO(3.13): Remove aiohttp.helpers once https://github.com/python/cpython/pull/106771
 # is available in all supported cpython versions
 exclude-modules = "(^aiohttp\\.helpers)"
+
+[tool.black]
+# TODO: Remove when project metadata is moved here.
+# Black can read the value from [project.requires-python].
+target-version = ["py39", "py310", "py311", "py312"]
```

Comment:
```suggestion
target-version = ["py39", "py310", "py311", "py312", "py313"]
```

Reply:
To make it work we also should upgrade pre-commit version in requirements and constraints.
I prefer don't touch it, a chance to get merge-conflict because of 3.13 specific formatting is very low, at least I don't see any specific code in generated diffs.